### PR TITLE
`AttachmentsFeatureElementView` - Capture new photos in JPEG

### PR DESCRIPTION
**Pull request by dfeinzimer**
Apollo 776

### CODE REVIEW

File: Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentCameraController.swift [hunk 1]
```diff
@@ -60,8 +60,8 @@ final class CameraControllerCoordinator: NSObject, UIImagePickerControllerDelega
     ) {
         parent.importState = .importing
         if let image = info[UIImagePickerController.InfoKey.originalImage] as? UIImage {
-            if let pngData = image.pngData() {
-                parent.importState = .finalizing(AttachmentImportData(contentType: .png, data: pngData))
+            if let jpegData = image.jpegData(compressionQuality: 0.9) {
```
Comment (mhdostal) [inline Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentCameraController.swift:63]: Why 0.9?

**Review by mhdostal — COMMENTED** [review]
Just the one question.

File: Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentCameraController.swift (hunk 1 above)
Comment (dfeinzimer) [reply Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentCameraController.swift:63]: Mark B noted in his issue he's getting images under 3 MB using the native camera app. With full 1.0 lossless, I was seeing ~3.5 MB. 0.9 was putting me right under 3 MB.

I'm open to whichever number you'd like to proceed with.

Comment (mhdostal) [reply Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentCameraController.swift:63]: That's fine, just curious. Users can change it if they want... :-)

**Review by mhdostal — APPROVED** [review]

**Pull request merged**